docs/qs_intro.md

```diff
@@ -1,61 +1,57 @@
+The most advanced and featureful is the Buildroot image. If you want the full GUI, this is currently your only choice. _Please note that this image is older and running on OVOS 0.0.6. A new Buildroot image is in development but not yet available._
 
-### Decompress the image
-Some image writing methods, `dd`, may require your file be decompressed.  Others, BalenaEtcher for example, can use a compressed image.
-The Buildroot image is compressed in `.xz` format and the raspbian image is in `.zip` format.
+- [RPi4-64](https://drive.google.com/file/d/1PUtNXfZ5jMUlVAgyN-KXPdVdX6r51eBw/view?usp=share_link)
 
-**Windows**
+[Instructions on flashing the image can be found here](flashing_images.md).
 
-Use `winzip` or `7-zip` to decompress the image.
+#### Docker on Raspberry Pi
 
-**Linux**
+The Docker version of OVOS works well on a Raspberry Pi with any distro. It also has a minimal GUI available. For more information, see the [Docker instructions](#docker).
 
-Use `gunzip` to decompress `.xz` compressed images and `unzip` to decompress `.zip` images.
+## Docker
 
-The resulting file should end in `.img` and is now ready to flash to a device.
+For most users, the easiest way to get started with OVOS is to use Docker. Docker is a containerization platform that allows you to run applications in a sandboxed environment. This means that you can run OVOS on any operating system that supports Docker, including Windows, MacOS, and Linux. Podman is also supported.
 
-### Flashing Software
+The new OVOS buildroot image (under construction) uses Podman to manage the OVOS services, allowing for easy setup and upgrades.
 
-Flashing your image to your SD card or USB drive is not different from flashing any other image. For the non-technical users we advise to use the flashing utility, [Raspberry Pi Imager](https://www.raspberrypi.com/software/) from the Raspberry Pi Foundation.  It is available for Windows, Mac OS, and Linux.
+[https://github.com/OpenVoiceOS/ovos-docker](https://github.com/OpenVoiceOS/ovos-docker)
 
-- Start up the Raspberry Pi Imager.  On Linux, start Raspberry Pi Imager with "**sudo** raspi-imager".
-- For "Choose OS", select "Use custom" and select the OVOS/PI image file downloaded from the OpenVoiceOS Downloads Site.
-- For "Choose Storage", select your removable boot media, probably something like "Internal SD Card Reader".
-- Then select "Write". Do not click the cog. Do not change the username. Username is ovos and is built in. Do not enter WiFi credentials, you will add them at first boot.
+The README at the link above explains how to set up OVOS on Docker for multiple different architectures and operating systems. New images are built nightly.
 
-Upon completion, you should have a bootable SD card or USB drive.
+**NOTE** The Docker images are all running alpha versions of 0.0.7 until OVOS 0.0.8 is released. At that point, users will have the option to choose between alpha or stable versions.
```

Comment:
This is a good note for raspOVOS also.  Every image pulls from github right now until a stable version is out